src/pages/Home/Home.scss

```diff
@@ -0,0 +1,32 @@
+@use "../../styles/variables" as *;
+
+.homepage {
+  margin: 2.5rem;
+}
+
+button {
```

Comment:
Remove this the `<Button>` has a prop to allow you to do this kind of CSS tweaks.